### ISSUE TITLE
bug: fails for "@typescript-eslint/eslint-plugin:~5.3.0"

### ISSUE THREAD

**Issue by pranav-js** [stencil-community/stencil-eslint#68]
### Prerequisites

- [X] I have read the [Contributing Guidelines](https://github.com/ionic-team/stencil/blob/main/.github/CONTRIBUTING.md).
- [X] I agree to follow the [Code of Conduct](https://github.com/ionic-team/stencil/blob/main/CODE_OF_CONDUCT.md).
- [X] I have searched for [existing issues](https://github.com/ionic-team/stencil-eslint/issues) that already report this problem, without success.

### Stencil ESLint Version

latest

### Current Behavior

```sh
╰─ npm i --save-dev @stencil/eslint-plugin
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: zwitch-frontend@2.0.0
npm ERR! Found: @typescript-eslint/eslint-plugin@5.3.1
npm ERR! node_modules/@typescript-eslint/eslint-plugin
npm ERR!   dev @typescript-eslint/eslint-plugin@"~5.3.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @typescript-eslint/eslint-plugin@"<5 && ^4.0.0" from @stencil/eslint-plugin@0.4.0
npm ERR! node_modules/@stencil/eslint-plugin
npm ERR!   dev @stencil/eslint-plugin@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See /Users/pranav.sarda/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/pranav.sarda/.npm/_logs/2021-12-28T11_18_35_507Z-debug.log
```

### Expected Behavior

should install without conflicitng peer dependencies. 

### Steps to Reproduce

I have a monorepo setup with nx and nxeext/stenciljs. Nx setup with 5.3.0 version of eslint hence causing conflict when trying to install this as -D

### Code Reproduction URL

just try with fresh nx repo + stencil

**Comment by pranav-js** [comment]
fixed with ```npm i --save-dev @stencil/eslint-plugin --legacy-peer-deps``` for now

**Comment by raphaelpor** [comment]
Fixed with 0.5.0.
